src/test/scala/beam/agentsim/agents/PersonWithPersonalVehiclePlanSpec.scala

```diff
@@ -680,6 +680,8 @@ class PersonWithPersonalVehiclePlanSpec
       personEntersVehicleEvents.expectMsgType[PersonEntersVehicleEvent]
 
       expectMsgType[CompletionNotice]
+
+      // TODO: Testing last resort vehicle creation
```

Comment:
Is there an issue for that?
If not - please, create one.

Reply:
This is related to another PR. I think it should be separate from this PR